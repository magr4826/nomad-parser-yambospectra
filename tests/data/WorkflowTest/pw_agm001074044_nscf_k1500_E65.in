&CONTROL
  calculation = 'nscf',
  disk_io = 'medium',
  outdir = 'out/',
  prefix = 'agm001074044',
  pseudo_dir = '/usr/scratch4/magr4826/Promotion/Python/Code/HT_RPA/pseudo/SG15',
/
&SYSTEM
  celldm(1) = 8.630314263567293,
  celldm(3) = 3.3923767235611275,
  degauss = 0.0004593661607577953,
  ecutwfc = 65,
  force_symmorphic = .TRUE.,
  ibrav = 7,
  nbnd = 85,
  nosym = .FALSE.,
  occupations = 'fixed',
  smearing = 'gaussian',
  nat = 7,
  ntyp = 3,
/
&ELECTRONS
  conv_thr = 1d-10,
  diago_full_acc = .TRUE.,
  diago_thr_init = 5d-06,
  diagonalization = 'david',
/
&IONS
/
&CELL
/
ATOMIC_SPECIES
  As  74.9216 As.upf
  Ca  40.0780 Ca.upf
  O  15.9994 O.upf
ATOMIC_POSITIONS crystal
  Ca 0.00000000 0.82826505 0.82826497
  Ca 0.00000000 0.17173503 0.17173503
  Ca 0.49999996 0.50000004 0.00000000
  Ca 0.50000004 0.00000000 0.49999996
  As 0.00000000 0.63722129 0.63722122
  As 0.00000000 0.36277878 0.36277878
  O 0.00000000 0.00000000 0.00000000
K_POINTS crystal
216
0.015625 0.03125 0.046875 1.0
0.015625 0.03125 0.21354167 1.0
0.015625 0.03125 0.38020833 1.0
0.015625 0.03125 0.546875 1.0
0.015625 0.03125 0.71354167 1.0
0.015625 0.03125 0.88020833 1.0
0.015625 0.19791667 0.046875 1.0
0.015625 0.19791667 0.21354167 1.0
0.015625 0.19791667 0.38020833 1.0
0.015625 0.19791667 0.546875 1.0
0.015625 0.19791667 0.71354167 1.0
0.015625 0.19791667 0.88020833 1.0
0.015625 0.36458333 0.046875 1.0
0.015625 0.36458333 0.21354167 1.0
0.015625 0.36458333 0.38020833 1.0
0.015625 0.36458333 0.546875 1.0
0.015625 0.36458333 0.71354167 1.0
0.015625 0.36458333 0.88020833 1.0
0.015625 0.53125 0.046875 1.0
0.015625 0.53125 0.21354167 1.0
0.015625 0.53125 0.38020833 1.0
0.015625 0.53125 0.546875 1.0
0.015625 0.53125 0.71354167 1.0
0.015625 0.53125 0.88020833 1.0
0.015625 0.69791667 0.046875 1.0
0.015625 0.69791667 0.21354167 1.0
0.015625 0.69791667 0.38020833 1.0
0.015625 0.69791667 0.546875 1.0
0.015625 0.69791667 0.71354167 1.0
0.015625 0.69791667 0.88020833 1.0
0.015625 0.86458333 0.046875 1.0
0.015625 0.86458333 0.21354167 1.0
0.015625 0.86458333 0.38020833 1.0
0.015625 0.86458333 0.546875 1.0
0.015625 0.86458333 0.71354167 1.0
0.015625 0.86458333 0.88020833 1.0
0.18229167 0.03125 0.046875 1.0
0.18229167 0.03125 0.21354167 1.0
0.18229167 0.03125 0.38020833 1.0
0.18229167 0.03125 0.546875 1.0
0.18229167 0.03125 0.71354167 1.0
0.18229167 0.03125 0.88020833 1.0
0.18229167 0.19791667 0.046875 1.0
0.18229167 0.19791667 0.21354167 1.0
0.18229167 0.19791667 0.38020833 1.0
0.18229167 0.19791667 0.546875 1.0
0.18229167 0.19791667 0.71354167 1.0
0.18229167 0.19791667 0.88020833 1.0
0.18229167 0.36458333 0.046875 1.0
0.18229167 0.36458333 0.21354167 1.0
0.18229167 0.36458333 0.38020833 1.0
0.18229167 0.36458333 0.546875 1.0
0.18229167 0.36458333 0.71354167 1.0
0.18229167 0.36458333 0.88020833 1.0
0.18229167 0.53125 0.046875 1.0
0.18229167 0.53125 0.21354167 1.0
0.18229167 0.53125 0.38020833 1.0
0.18229167 0.53125 0.546875 1.0
0.18229167 0.53125 0.71354167 1.0
0.18229167 0.53125 0.88020833 1.0
0.18229167 0.69791667 0.046875 1.0
0.18229167 0.69791667 0.21354167 1.0
0.18229167 0.69791667 0.38020833 1.0
0.18229167 0.69791667 0.546875 1.0
0.18229167 0.69791667 0.71354167 1.0
0.18229167 0.69791667 0.88020833 1.0
0.18229167 0.86458333 0.046875 1.0
0.18229167 0.86458333 0.21354167 1.0
0.18229167 0.86458333 0.38020833 1.0
0.18229167 0.86458333 0.546875 1.0
0.18229167 0.86458333 0.71354167 1.0
0.18229167 0.86458333 0.88020833 1.0
0.34895833 0.03125 0.046875 1.0
0.34895833 0.03125 0.21354167 1.0
0.34895833 0.03125 0.38020833 1.0
0.34895833 0.03125 0.546875 1.0
0.34895833 0.03125 0.71354167 1.0
0.34895833 0.03125 0.88020833 1.0
0.34895833 0.19791667 0.046875 1.0
0.34895833 0.19791667 0.21354167 1.0
0.34895833 0.19791667 0.38020833 1.0
0.34895833 0.19791667 0.546875 1.0
0.34895833 0.19791667 0.71354167 1.0
0.34895833 0.19791667 0.88020833 1.0
0.34895833 0.36458333 0.046875 1.0
0.34895833 0.36458333 0.21354167 1.0
0.34895833 0.36458333 0.38020833 1.0
0.34895833 0.36458333 0.546875 1.0
0.34895833 0.36458333 0.71354167 1.0
0.34895833 0.36458333 0.88020833 1.0
0.34895833 0.53125 0.046875 1.0
0.34895833 0.53125 0.21354167 1.0
0.34895833 0.53125 0.38020833 1.0
0.34895833 0.53125 0.546875 1.0
0.34895833 0.53125 0.71354167 1.0
0.34895833 0.53125 0.88020833 1.0
0.34895833 0.69791667 0.046875 1.0
0.34895833 0.69791667 0.21354167 1.0
0.34895833 0.69791667 0.38020833 1.0
0.34895833 0.69791667 0.546875 1.0
0.34895833 0.69791667 0.71354167 1.0
0.34895833 0.69791667 0.88020833 1.0
0.34895833 0.86458333 0.046875 1.0
0.34895833 0.86458333 0.21354167 1.0
0.34895833 0.86458333 0.38020833 1.0
0.34895833 0.86458333 0.546875 1.0
0.34895833 0.86458333 0.71354167 1.0
0.34895833 0.86458333 0.88020833 1.0
0.515625 0.03125 0.046875 1.0
0.515625 0.03125 0.21354167 1.0
0.515625 0.03125 0.38020833 1.0
0.515625 0.03125 0.546875 1.0
0.515625 0.03125 0.71354167 1.0
0.515625 0.03125 0.88020833 1.0
0.515625 0.19791667 0.046875 1.0
0.515625 0.19791667 0.21354167 1.0
0.515625 0.19791667 0.38020833 1.0
0.515625 0.19791667 0.546875 1.0
0.515625 0.19791667 0.71354167 1.0
0.515625 0.19791667 0.88020833 1.0
0.515625 0.36458333 0.046875 1.0
0.515625 0.36458333 0.21354167 1.0
0.515625 0.36458333 0.38020833 1.0
0.515625 0.36458333 0.546875 1.0
0.515625 0.36458333 0.71354167 1.0
0.515625 0.36458333 0.88020833 1.0
0.515625 0.53125 0.046875 1.0
0.515625 0.53125 0.21354167 1.0
0.515625 0.53125 0.38020833 1.0
0.515625 0.53125 0.546875 1.0
0.515625 0.53125 0.71354167 1.0
0.515625 0.53125 0.88020833 1.0
0.515625 0.69791667 0.046875 1.0
0.515625 0.69791667 0.21354167 1.0
0.515625 0.69791667 0.38020833 1.0
0.515625 0.69791667 0.546875 1.0
0.515625 0.69791667 0.71354167 1.0
0.515625 0.69791667 0.88020833 1.0
0.515625 0.86458333 0.046875 1.0
0.515625 0.86458333 0.21354167 1.0
0.515625 0.86458333 0.38020833 1.0
0.515625 0.86458333 0.546875 1.0
0.515625 0.86458333 0.71354167 1.0
0.515625 0.86458333 0.88020833 1.0
0.68229167 0.03125 0.046875 1.0
0.68229167 0.03125 0.21354167 1.0
0.68229167 0.03125 0.38020833 1.0
0.68229167 0.03125 0.546875 1.0
0.68229167 0.03125 0.71354167 1.0
0.68229167 0.03125 0.88020833 1.0
0.68229167 0.19791667 0.046875 1.0
0.68229167 0.19791667 0.21354167 1.0
0.68229167 0.19791667 0.38020833 1.0
0.68229167 0.19791667 0.546875 1.0
0.68229167 0.19791667 0.71354167 1.0
0.68229167 0.19791667 0.88020833 1.0
0.68229167 0.36458333 0.046875 1.0
0.68229167 0.36458333 0.21354167 1.0
0.68229167 0.36458333 0.38020833 1.0
0.68229167 0.36458333 0.546875 1.0
0.68229167 0.36458333 0.71354167 1.0
0.68229167 0.36458333 0.88020833 1.0
0.68229167 0.53125 0.046875 1.0
0.68229167 0.53125 0.21354167 1.0
0.68229167 0.53125 0.38020833 1.0
0.68229167 0.53125 0.546875 1.0
0.68229167 0.53125 0.71354167 1.0
0.68229167 0.53125 0.88020833 1.0
0.68229167 0.69791667 0.046875 1.0
0.68229167 0.69791667 0.21354167 1.0
0.68229167 0.69791667 0.38020833 1.0
0.68229167 0.69791667 0.546875 1.0
0.68229167 0.69791667 0.71354167 1.0
0.68229167 0.69791667 0.88020833 1.0
0.68229167 0.86458333 0.046875 1.0
0.68229167 0.86458333 0.21354167 1.0
0.68229167 0.86458333 0.38020833 1.0
0.68229167 0.86458333 0.546875 1.0
0.68229167 0.86458333 0.71354167 1.0
0.68229167 0.86458333 0.88020833 1.0
0.84895833 0.03125 0.046875 1.0
0.84895833 0.03125 0.21354167 1.0
0.84895833 0.03125 0.38020833 1.0
0.84895833 0.03125 0.546875 1.0
0.84895833 0.03125 0.71354167 1.0
0.84895833 0.03125 0.88020833 1.0
0.84895833 0.19791667 0.046875 1.0
0.84895833 0.19791667 0.21354167 1.0
0.84895833 0.19791667 0.38020833 1.0
0.84895833 0.19791667 0.546875 1.0
0.84895833 0.19791667 0.71354167 1.0
0.84895833 0.19791667 0.88020833 1.0
0.84895833 0.36458333 0.046875 1.0
0.84895833 0.36458333 0.21354167 1.0
0.84895833 0.36458333 0.38020833 1.0
0.84895833 0.36458333 0.546875 1.0
0.84895833 0.36458333 0.71354167 1.0
0.84895833 0.36458333 0.88020833 1.0
0.84895833 0.53125 0.046875 1.0
0.84895833 0.53125 0.21354167 1.0
0.84895833 0.53125 0.38020833 1.0
0.84895833 0.53125 0.546875 1.0
0.84895833 0.53125 0.71354167 1.0
0.84895833 0.53125 0.88020833 1.0
0.84895833 0.69791667 0.046875 1.0
0.84895833 0.69791667 0.21354167 1.0
0.84895833 0.69791667 0.38020833 1.0
0.84895833 0.69791667 0.546875 1.0
0.84895833 0.69791667 0.71354167 1.0
0.84895833 0.69791667 0.88020833 1.0
0.84895833 0.86458333 0.046875 1.0
0.84895833 0.86458333 0.21354167 1.0
0.84895833 0.86458333 0.38020833 1.0
0.84895833 0.86458333 0.546875 1.0
0.84895833 0.86458333 0.71354167 1.0
0.84895833 0.86458333 0.88020833 1.0

