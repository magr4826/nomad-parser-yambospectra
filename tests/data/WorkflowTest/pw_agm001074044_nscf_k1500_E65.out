
     Program PWSCF v.7.1 starts on 14Jun2024 at 13:27:17 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on    16 processors

     MPI processes distributed on     1 nodes
     149900 MiB available memory on the printing compute node when the environment starts
 
     Reading input from pw_agm001074044_nscf_k1500_E65.in
Warning: card &IONS ignored
Warning: card / ignored
Warning: card &CELL ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4

     Atomic positions and unit cell read from directory:
     out/agm001074044.save/
     Atomic positions from file used, from input discarded
 
 
     R & G space division:  proc/nbgrp/npool/nimage =      16
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         170     170     53                 4821     4821     847
     Max         171     171     54                 4822     4822     849
     Sum        2725    2725    851                77147    77147   13563
 
     Using Slab Decomposition
 


     bravais-lattice index     =            7
     lattice parameter (alat)  =       8.6303  a.u.
     unit-cell volume          =    1090.3198 (a.u.)^3
     number of atoms/cell      =            7
     number of atomic types    =            3
     number of electrons       =        56.00
     number of Kohn-Sham states=           85
     kinetic-energy cutoff     =      65.0000  Ry
     charge density cutoff     =     260.0000  Ry
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=   8.630314  celldm(2)=   0.000000  celldm(3)=   3.392377
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.500000  -0.500000   1.696188 )  
               a(2) = (   0.500000   0.500000   1.696188 )  
               a(3) = (  -0.500000  -0.500000   1.696188 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000 -1.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.294779 )  
               b(3) = ( -1.000000  0.000000  0.294779 )  


     PseudoPot. # 1 for As read from file:
     /usr/scratch4/magr4826/Promotion/Python/Code/HT_RPA/pseudo/SG15/As.upf
     MD5 check sum: a2ff92a4799c87c945fff9e6739df4dd
     Pseudo is Norm-conserving, Zval =  5.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of  602 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     PseudoPot. # 2 for Ca read from file:
     /usr/scratch4/magr4826/Promotion/Python/Code/HT_RPA/pseudo/SG15/Ca.upf
     MD5 check sum: b43c970dc090f2312d256360aabdb6f1
     Pseudo is Norm-conserving, Zval = 10.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of  602 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     PseudoPot. # 3 for O  read from file:
     /usr/scratch4/magr4826/Promotion/Python/Code/HT_RPA/pseudo/SG15/O.upf
     MD5 check sum: e36c48deb68f4e355fffe4602be2c42c
     Pseudo is Norm-conserving, Zval =  6.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of  602 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1

     atomic species   valence    mass     pseudopotential
        As             5.00    74.92160     As( 1.00)
        Ca            10.00    40.07800     Ca( 1.00)
        O              6.00    15.99940     O ( 1.00)

     16 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Ca  tau(   1) = (   0.0000000   0.0000000   2.8097869  )
         2           Ca  tau(   2) = (   0.0000000   0.0000000   0.5825899  )
         3           Ca  tau(   3) = (   0.5000000   0.0000000   1.6961884  )
         4           Ca  tau(   4) = (   0.0000000  -0.5000000   1.6961884  )
         5           As  tau(   5) = (   0.0000000   0.0000000   2.1616946  )
         6           As  tau(   6) = (   0.0000000   0.0000000   1.2306823  )
         7           O   tau(   7) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=   216

     Number of k-points >= 100: set verbosity='high' to print them.

     Dense  grid:    77147 G-vectors     FFT dimensions: (  81,  81,  81)

     Estimated max dynamical RAM per process >      13.79 MB

     Estimated total dynamical RAM >     220.58 MB

     The potential is recalculated from file :
     out/agm001074044.save/charge-density

     Starting wfcs are random

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  5.00E-06,  avg # of iterations = 22.0

     total cpu time spent up to now is      237.8 secs

     End of band structure calculation

     Number of k-points >= 100: set verbosity='high' to print the bands.

     highest occupied, lowest unoccupied level (ev):     7.0317    8.3062

     Writing all to output data dir out/agm001074044.save/
 
     init_run     :      0.12s CPU      0.20s WALL (       1 calls)
     electrons    :    234.97s CPU    237.35s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.05s CPU      0.05s WALL (       1 calls)
     hinit0       :      0.03s CPU      0.09s WALL (       1 calls)

     Called by electrons:
     c_bands      :    234.97s CPU    237.35s WALL (       1 calls)
     v_of_rho     :      0.01s CPU      0.02s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.05s CPU      0.08s WALL (     216 calls)
     init_us_2:cp :      0.05s CPU      0.08s WALL (     216 calls)
     cegterg      :    220.15s CPU    221.99s WALL (     364 calls)

     Called by *egterg:
     cdiaghg      :     29.84s CPU     29.98s WALL (    4960 calls)
     h_psi        :    183.14s CPU    184.73s WALL (    5324 calls)
     g_psi        :      0.29s CPU      0.29s WALL (    4744 calls)

     Called by h_psi:
     h_psi:calbec :      3.62s CPU      3.68s WALL (    5324 calls)
     vloc_psi     :    175.51s CPU    177.02s WALL (    5324 calls)
     add_vuspsi   :      3.45s CPU      3.47s WALL (    5324 calls)

     General routines
     calbec       :      3.61s CPU      3.67s WALL (    5324 calls)
     fft          :      0.01s CPU      0.07s WALL (      10 calls)
     fftw         :    165.50s CPU    166.94s WALL (  527228 calls)
     davcio       :      0.11s CPU      0.53s WALL (     432 calls)
 
     Parallel routines
 
     PWSCF        :   3m58.37s CPU   4m 6.28s WALL

 
   This run was terminated on:  13:31:24  14Jun2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
